client/gutenberg/extensions/map-block/edit.js

```diff
@@ -0,0 +1,144 @@
+/** @format */
+
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+import { Component, createRef, Fragment } from '@wordpress/element';
+
+import { IconButton, PanelBody, Toolbar } from '@wordpress/components';
+
+import {
+	InspectorControls,
+	BlockControls,
+	BlockAlignmentToolbar,
+	PanelColorSettings,
+} from '@wordpress/editor';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import { CONFIG } from './config.js';
+import AddPoint from './add-point';
+import Locations from './locations';
+import Map from './component.js';
+import MapThemePicker from './map-theme-picker';
+
+/**
+ * Module variables
+ */
+
+class MapEdit extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			addPointVisibility: false,
+		};
+		this.mapRef = createRef();
+		this.addPoint = this.addPoint.bind( this );
```

Comment:
Seems like we could remove all the `bind` calls here if we use arrow functions.

Reply:
Done https://github.com/Automattic/wp-calypso/pull/27836/commits/92aa832fd43ffac9ffdbc63ec43c68a9d9346760